std/assembly/util/string.ts

```diff
@@ -770,7 +797,7 @@ export function strtod(str: string): f64 {
       load<u64>(ptr, 0) == 0x690066006E0049 && // ifnI
       load<u64>(ptr, 8) == 0x7900740069006E    // ytin
     ) {
-      return copysign<f64>(Infinity, sign);
+      return Infinity * sign;
```

Comment:
What's the difference here? Just curious :)

Reply:
just smaller in term of codesize

Reply:
Oh, is it? Would expect both binary ops having the same size in a binary.

Reply:
Ah, yeah. Same size. But `mul` simpler for codegenerators.

Reply:
https://godbolt.org/z/SkDktV

Reply:
Do you know if there's a difference here on specific processor architectures? Would be surprised if `copysign` existed if it wasn't preferrable over `mul` in certain scenarios.

Reply:
`copysign` always translate to several operations in all known archs. Also `mul` may be little slower but this case is really rare in any case

Reply:
From another perspective: So, would it be feasible for Binaryen to optimize from `mul` to `copysign`?

Reply:
I don't think so, it's not reduce codesize but require extra analysis and not always safe